DependencyInjection/BazingaGeocoderExtension.php

```diff
@@ -106,7 +106,11 @@ public function configureProviderPlugins(ContainerBuilder $container, array $con
     {
         $plugins = [];
         foreach ($config['plugins'] as $plugin) {
-            $plugins[] = $plugin['id'];
+            if (isset($plugin['reference'])) {
+                $plugins[] = $plugin['reference']['id'];
+            } else {
+                $plugins[] = $plugin['id'];
```

Comment:
This is super weird. I cannot make any test that executes this line... Has this ever been working?

Reply:
I have nothing to do with that ;) But IMO there should always be a reference entry now.

Reply:
> I have nothing to do with that ;) 

Yeah, I know. 

> But IMO there should always be a reference entry now.

Great. Could you finish this PR and I'll be happy to merge?